Documentation update for sap-converged-cloud (#309)
kyma-project/kyma-environment-broker: docs/contributor/03-10-hyperscaler-account-pool.md

For a certain type of SAP BTP, Kyma runtimes, KEB can use the same credentials for multiple tenants.
In such a case, the Secret with credentials must be labeled differently by adding the **shared** label set to `true`. Shared credentials will not be assigned to any tenant.

> The Secret with credentials can be shared by multiple tenants i.e. many shoots (Shoot resources) can refer to the same Secret (this reference is represented by the SecretBinding resource).

Reviewer: ```suggestion
Multiple tenants can share the Secret with credentials. That is, many shoots (Shoot resources) can refer to the same Secret. This reference is represented by the SecretBinding resource.
```

    shared: "true"
```

> ### Shared Credentials for sap-converged-cloud plan

Reviewer: ```suggestion
### Shared Credentials for `sap-converged-cloud` Plan
```

> For a sap-converged-cloud plan each region is treated as a separate hyperscaler. Hence Secrets are labeled with **openstack_{region name}** e.g. **openstack_eu-de-1**.

Reviewer: ```suggestion
For the `sap-converged-cloud` plan, each region is treated as a separate hyperscaler. Hence, Secrets are labeled with **openstack_{region name}**, for example, **openstack_eu-de-1**.
```
